### PR TITLE
Use of checkers on docker_cli_history_test.go

### PR DESCRIPTION
**Pull request by jecarey**
Signed-off-by: James Carey jecarey@us.ibm.com

### CODE REVIEW

File: integration-cli/docker_cli_history_test.go [hunk 1]
```diff
@@ -54,10 +53,7 @@ RUN echo "Z"`,
 	for i := 0; i < 26; i++ {
 		echoValue := fmt.Sprintf("echo \"%s\"", expectedValues[i])
 		actualValue := actualValues[i]
-
-		if !strings.Contains(actualValue, echoValue) {
-			c.Fatalf("Expected layer \"%s\", but was: %s", expectedValues[i], actualValue)
-		}
+		c.Assert(actualValue, checker.Contains, echoValue, check.Commentf("Expected layer \"%s\", but was: %s", expectedValues[i], actualValue))
```
Comment (vdemeester) [inline integration-cli/docker_cli_history_test.go]: This comment is *almost* not needed (as it repeats what's checked).

File: integration-cli/docker_cli_history_test.go [hunk 2]
```diff
@@ -91,10 +85,7 @@ func (s *DockerSuite) TestHistoryImageWithComment(c *check.C) {
 	out, _ := dockerCmd(c, "history", name)
 	outputTabs := strings.Fields(strings.Split(out, "\n")[1])
 	actualValue := outputTabs[len(outputTabs)-1]
-
-	if !strings.Contains(actualValue, comment) {
-		c.Fatalf("Expected comments %q, but found %q", comment, actualValue)
-	}
+	c.Assert(actualValue, checker.Contains, comment, check.Commentf("Expected comments %q, but found %q", comment, actualValue))
```
Comment (vdemeester) [inline integration-cli/docker_cli_history_test.go]: Same here :P

File: integration-cli/docker_cli_history_test.go [hunk 3]
```diff
@@ -121,7 +112,7 @@ func (s *DockerSuite) TestHistoryHumanOptionTrue(c *check.C) {
 	out, _ := dockerCmd(c, "history", "--human=true", "busybox")
 	lines := strings.Split(out, "\n")
 	sizeColumnRegex, _ := regexp.Compile("SIZE +")
-	humanSizeRegex, _ := regexp.Compile("^\\d+.*B$") // Matches human sizes like 10 MB, 3.2 KB, etc
+	humanSizeRegexRaw := "^\\d+.*B$" // Matches human sizes like 10 MB, 3.2 KB, etc
```
Comment (vdemeester) [inline integration-cli/docker_cli_history_test.go]: Hum the checker `Matches` already add `^` and `$` (https://github.com/go-check/check/blob/v1/checkers.go#L307) so I guess they are not needed — but if it works like this it feels ok too :sweat_smile: .

File: integration-cli/docker_cli_history_test.go (hunk 1 above)
Comment (jecarey) [inline integration-cli/docker_cli_history_test.go]: I looked at the output when it fails and I agree the comment really isn't needed, so I removed it.

File: integration-cli/docker_cli_history_test.go (hunk 2 above)
Comment (jecarey) [inline integration-cli/docker_cli_history_test.go]: Did this one too

File: integration-cli/docker_cli_history_test.go (hunk 3 above)
Comment (jecarey) [inline integration-cli/docker_cli_history_test.go]: Thanks for pointing that out.   I removed them, since even though it works, it is confusing.

**Comment by vdemeester** [comment]
LGTM 🐹

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**